src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -239,21 +258,42 @@ object AddDatasetMetadataToDeposit {
     createSchemedMetadata(dataset, composedSubjectFields, "DC_SUBJECT", "DC_SUBJECT_SCHEME")
   }
 
+  /*
+    qualifier   link   title   valid
+        1        1       1       0
+        1        1       0       1
+        1        0       1       1
+        1        0       0       0
+        0        1       1       0
+        0        1       0       1
+        0        0       1       1
+        0        0       0       1
+
+    observation: if the qualifier is present, either DCX_RELATION_LINK or DCX_RELATION_TITLE must be defined
+                 if the qualifier is not defined, DCX_RELATION_LINK and DCX_RELATION_TITLE must not both be defined
+   */
   def createRelations(dataset: Dataset): Seq[Elem] = {
-    dataset.rowsWithValuesFor(composedRelationFields).map { row =>
+    dataset.rowsWithValuesFor(composedRelationFields).map(row =>
       (row.get("DCX_RELATION_QUALIFIER"), row.get("DCX_RELATION_LINK"), row.get("DCX_RELATION_TITLE")) match {
         // @formatter:off
         case (Some(q), Some(l), _      ) => elem(s"dcterms:$q")(l)
         case (Some(q), None,    Some(t)) => elem(s"dcterms:$q")(t)
         case (None,    Some(l), _      ) => elem(s"dc:relation")(l)
         case (None,    None,    Some(t)) => elem(s"dc:relation")(t)
         case _                           =>
-          // TODO this case needs to be checked to not occur in the preconditions
-          // (see also comment in https://github.com/DANS-KNAW/easy-split-multi-deposit/commit/dbda6cc2b78f93196be62b323a988e3781cb6926#diff-efd2dc8d9655ba9c6b577f13dd66627bR32)
+          // all other cases are checked not to occur in the preconditions
           throw new IllegalArgumentException("preconditions should have reported this as an error")
         // @formatter:on
-      }
-    }
+      })
+  }
+
+  def createSurrogateRelation(dataset: Dataset): Option[Elem] = {
+    CreateSpringfieldActions.getSpringfieldPath(dataset, 0)
+      .map(path =>
+        // @formatter:off
+        <ddm:relation scheme="STREAMING_SURROGATE_RELATION">{path}</ddm:relation>
```

Comment:
the STREAMING_SURROGATE_RELATION has the presentation as the final variable. this is not yet present during this stage, so it should be a placeholder